### DISCUSSION TITLE
Add Course to medium

### DISCUSSION THREAD

**Discussion by bmaluijb** [Podcastindex-org/podcast-namespace#577] [Enhancement Proposal]
The medium tag could use a "course" value, that indicates that the feed is a course.

**Comment by daveajones** [comment]
cc: @agates

**Comment by agates** [comment]
You mean like `<podcast:medium>course</podcast:medium>`?

What would the format be and how should applications interpret it in their user interface?

What is an established precedent as an example (pluralisight, udemy)?

Is it audio, video, or some kind of interactive format?

**Comment by bmaluijb** [comment]
Yeah, exactly that format. So course as the value.

This would be an audio course. Listenable is an example of audio only courses. I can also imagine lectures to fall into this category.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by daveajones** [comment]
Hmm. Lots of courses could be video also. I would hate to limit it. I like the idea a lot, but would want to be able to specify both.

**Comment by agates** [comment]
Yeah I was thinking it might not need to be limited to a specific format, I've seen courses that combine both even.  I'd like to at least list an established precedent for documentation purposes, and any one of the "courses" type of sites probably work.

Also wondering if someone already has plans for this use case

**Comment by bmaluijb** [comment]
I myself have 3 courses that I want to deploy in an audio format.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by agates** [comment]
@daveajones I'm for it, I can get it added to podping-hivewriter but I'd like to wait until we decide on channels/publishers first just too reduce the number of releases there.

**Comment by samsethi** [comment]
Do we have any example feeds to test against? We have added medium=course and now look for it.  We will change the UI to say Episode > Lessons. or should that be modules?? Any other UI/UX changes we could make.

**Comment by bmaluijb** [comment]
I have one of mine: https://serve.podhome.fm/rss/a1cf56c2-31e0-450b-ad6d-afe2ecbc9cc1

One thing I notice is that the normal sort order by publishdate might not be meaningful for a course. Perhaps a default on episode nr should be maintained when medium=course

**Comment by Marzal** [comment]
Just for the record, this change was formalized on [2024-06-07](https://github.com/Podcastindex-org/podcast-namespace/commit/437559f971d207d5706e43e6c4469a2909add78e)

Announcement:
https://github.com/Podcastindex-org/podcast-namespace/discussions/642
